### ISSUE TITLE
학습 시나리오 2 테스트

### ISSUE THREAD

**Issue by TukBang** [TukBang/Pet-care-app#4]
Model 1: A1 ~ A3 (3-CLASS), EfficientNetV2_s
Model 2: A4 ~ A6 (3-CLASS), EfficientNetV2_s
Preprocessing: ratio-and-zero-padding, interpolation: LANCZOS4
batch_size: 64, 128
optimizer = Adam
learning rate: 1e-4
criterion = CrossEntropyLoss

레이블 특징이 비슷한 클래스끼리 학습을 진행

학습 시나리오 1에서
검증 데이터의 정확도는 63% AUC-ROC 0.88

**Comment by TukBang** [comment]
learning rate: 1e-4에서 로스가 변하지 않고 진동하는 상황이 발생하여
5e-3으로 변경

**Comment by TukBang** [comment]
![image](https://user-images.githubusercontent.com/121234327/226604636-ef82371c-4d67-42e9-857c-5272e5218716.png)

[Model 1]: batch_size 128
1. epoch 50부터 과적합 발생
2. 정확도는 65%로 시나리오 1보다 2% 상승
3. 시나리오 1보다 눈에 띄게 낮은 loss 기록

**Comment by TukBang** [comment]
[Model 1]: batch_size 128
![image](https://user-images.githubusercontent.com/121234327/226606388-32838ac4-dc96-4492-86e0-799e2a3d515b.png)
AUC per class: [0.8461344174631805, 0.8264197162005225, 0.8044148762790837]

**Comment by TukBang** [comment]
[Model 1]: batch_size 128

              precision    recall  f1-score   support

          A1       0.60      0.72      0.66      1087
          A2       0.69      0.66      0.68      1331
          A3       0.67      0.60      0.63      1317

    accuracy                           0.65      3735

학습 시나리오 6-CLASS보다
A1은 precision 1%, recall 16%, f1-score 9%가 높아짐
A2는 precision 7%, f1-score 3%가 높아지고, recall 1% 떨어짐
A3는 precision 14%, f1-score 2%가 높아지고, recall 9% 떨어짐

전체적인 수치로 봤을 때 모두 60%가 넘고, 학습시나리오 1 6-CLASS보다 높은 결과

**Comment by TukBang** [comment]
[Model 1]: batch_size 64
[Model 2]: batch_size 128, 64
진행 중

**Comment by TukBang** [comment]
[Model 2]: batch_size 128
![graph](https://user-images.githubusercontent.com/121234327/226840333-d808e4ee-f4c3-4d01-b768-83433b7a5d00.png)
![AUC-ROC curve](https://user-images.githubusercontent.com/121234327/226840360-566ab4bb-9967-4990-a15e-a0c93a469ad6.png)

              precision    recall  f1-score   support

          A4       0.63      0.77      0.70      1092
          A5       0.70      0.55      0.61      1056
          A6       0.81      0.79      0.80       762

    accuracy                           0.70      2910

**Comment by TukBang** [comment]
[Model 2]: batch_size 128

학습 시나리오 6-CLASS보다
A4는 recall 14%, f1-score 7%가 높아짐
A5는 precision 7%, recall 15%, f1-score 12%가 높아짐
A6는 precision 3%, recall 2%, f1-score 3%가 떨어짐

3 클래스로 나누어 학습하는 것이 6 클래스로 한번에 학습하는 것보다
성능이 많이 개선됨

특징으로 A6는 기존 6 클래스에도 80%에 도달,
3 클래스로 나누어도 비교했을 때 차이가 미미한 점을 고려해서

학습에 효과적인 방법일 수 있음
AUC-ROC 커브는 기존 6-CLASS보다 떨어진 수치

**Comment by TukBang** [comment]
batch_size 64는 batch_size 128보다 정확도, classification_report, AUC-ROC curve 모두 낮은 수치를 기록하여 포함하지 않음
